forch/device_report_server.py

```diff
@@ -21,6 +23,22 @@ def __init__(self, on_receiving_result):
         super().__init__()
         self._on_receiving_result = on_receiving_result
         self._logger = get_logger('drserver')
+        self._port_device_mapping = {}
+        self._port_events_listeners: Dict[str, List[Queue]] = {}
```

Comment:
stickler error... not even sure myself what the line is supposed to be doing!

Reply:
I tried to put a type definition there but that's generating a false positive from flake8.